modules/zip/doc_classes/ZIPPacker.xml

```diff
@@ -72,5 +77,17 @@
 		<constant name="APPEND_ADDINZIP" value="2" enum="ZipAppend">
 			Add new files to the existing zip archive at the given path.
 		</constant>
+		<constant name="COMPRESSION_DEFAULT" value="-1" enum="CompressionLevel">
+			Start a file with the default Deflate compression level ([code]6[/code]). This is a good compromise between speed and file size.
+		</constant>
+		<constant name="COMPRESSION_NONE" value="0" enum="CompressionLevel">
+			Start a file with no compression. This is also known as the "Store" compression mode and is the fastest method of packing files inside a ZIP archive. Consider using this mode for files that are already compressed (such as JPEG, PNG, MP3 or Ogg Vorbis files).
```

Comment:
```suggestion
			Start a file with no compression. This is also known as the "Store" compression mode and is the fastest method of packing files inside a ZIP archive. Consider using this mode for files that are already compressed (such as JPEG, PNG, MP3, or Ogg Vorbis files).
```
Please use [Oxford comma](https://docs.godotengine.org/en/latest/contributing/documentation/docs_writing_guidelines.html#use-the-oxford-comma-to-enumerate-anything).